src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/network/Network.kt

```diff
@@ -1,8 +1,11 @@
-package com.atlassian.performance.tools.awsinfrastructure.api
+package com.atlassian.performance.tools.awsinfrastructure.api.network
```

Comment:
We can make the move, because `com.atlassian.performance.tools.awsinfrastructure.api.Network` class is not released yet